docs/docs/cmd/spo/homesite/homesite-list.mdx

```diff
@@ -0,0 +1,107 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo homesite list
+
+Lists all the Home Sites
+
+## Usage
+
+```sh
+m365 spo homesite list [options]
+```
+
+## Options
+
+<Global />
+
+## Examples
+
+Lists all the Home Sites
+
+```sh
+m365 spo homesite list
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  {
+    "value": [
+      {
+        "Audiences": [
+          {
+            "Email": "ColumnSearchable@contoso.onmicrosoft.com",
+            "Id": "978b5280-4f80-47ea-a1db-b0d1d2fb1ba4",
+            "Title": "ColumnSearchable Members"
+          },
+          {
+            "Email": "contosoteam@contoso.onmicrosoft.com",
+            "Id": "21af775d-17b3-4637-94a4-2ba8625277cb",
+            "Title": "Contoso TeamR Members"
+          }
+        ],
+        "IsInDraftMode": false,
+        "IsVivaBackendSite": false,
+        "SiteId": "431d7819-4aaf-49a1-b664-b2fe9e609b63",
+        "TargetedLicenseType": 2,
+        "Title": "The Landing",
+        "Url": "https://contoso.sharepoint.com/sites/TheLanding",
+        "VivaConnectionsDefaultStart": true,
+        "WebId": "626c1724-8ac8-45d5-af87-c07c752fab75"
+      },
+      {
+        "Audiences": [],
+        "IsInDraftMode": false,
+        "IsVivaBackendSite": false,
+        "SiteId": "45d4a135-40e4-4571-8340-61d17fdfd58a",
+        "TargetedLicenseType": 0,
+        "Title": "Contoso Electronics",
+        "Url": "https://contoso.sharepoint.com/sites/contosoportal",
+        "VivaConnectionsDefaultStart": true,
+        "WebId": "9418e2a1-855c-4752-8dd4-48693f43b10a"
+      }
+    ]
+  }
+  ```
+
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+Url                                                              Title              
+---------------------------------------------------------------  -------------------
+https://reshmeeauckloo.sharepoint.com/sites/TheLanding           The Landing        
+https://reshmeeauckloo.sharepoint.com/sites/contosoportal        Contoso Electronics
+  ```
+
+  </TabItem>
+  <TabItem value="CSV">
+
+  ```csv
+  ```
+
+  </TabItem>
+  <TabItem value="Markdown">
+
+  ```md
+  m365 spo homesite list --output md      
+# spo homesite list 
+
+Date: 11/18/2024
+
+Property | Value
+---------|-------
+
```

Comment:
Seems like there is no output for CSV and MD.

Reply:
@milanholemans , I think there are properties which are not compatible with csv and md, I was hoping you can help me figure it out from the json output which ones are not compatible. Is it the audiences being returned as array?